### PR TITLE
Add automating sbs kb

### PR DESCRIPTION
**Pull request by jwebs1**

### CODE REVIEW

File: Backup/automating-sbs-on-api-v2.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+The following powershell script will iterate through each of the servers in the specified account alias and create the appropriate SBS policies based on the storage paths being used on the VMs. The correct operating system (OS) policies will be created for Windows and Linux respectively. The script will then iterate through the servers again and apply the appropriate policy based on the OS.
+
+There are a number of variables at the beginning of the script that you can manually edit before you execute the script. These are the account alias, the number of days the data will be retained, the backup frequency of the policy, and the storage region where the backups will be stored. These variables begin on line 44. The account alias has to be modified, the other variables are optional. If you don't modify the account alias, the script will prompt you to do so before it executes. If you don't edit the other variables within the script, they can be changed later in the Portal UI. An output file with results of the operation will be stored in: C:\users\public\CLC\SBSDeployment.
+
+## Using the Powershell Script
```
Comment (srveit) [inline Backup/automating-sbs-on-api-v2.md]: Change to `###`

File: Backup/automating-sbs-on-api-v2.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+14. Click on one of the policies to see the servers that it was applied to, as well as its settings.
+
+    ![sbspowershellportalpolicydetails.png](../images/sbspowershellportalpolicydetails.png)
+
+    **Note:** If you want to remove an SBS policy from a server, just click the server's name on this page and then the delete button.
+
+    ![sbspowershellportaldeletepolicy.png](../images/sbspowershellportaldeletepolicy.png)
+
+## Powershell Script
```
Comment (srveit) [inline Backup/automating-sbs-on-api-v2.md]: Change to `###`

File: Backup/automating-sbs-on-api-v2.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+    {
+        "$thisid is a $thisOS server"
+        applyPolicy($linPolicy)
+    } # end else
+} # end forEach
+
+ Write-Verbose -message "Operation complete. An output file with results will be stored at $filename." -verbose
+```
+
+## Want to Know More About the Simple Backup Service?
```
Comment (srveit) [inline Backup/automating-sbs-on-api-v2.md]: Change to `###`

**Comment by jwebs1** [comment]
@srveit updated headers

**Pull request merged**